src/stories/organisms-news.stories.js

```diff
@@ -0,0 +1,6 @@
+import Item_News from "../components/organisms/news/news.html";
+import "../components/organisms/news/news.scss";
+
+export default { title: "Organisms/News" };
+
+export const Events = () => Item_News;
```

Comment:
storybookでの表示がEventsになっているのでNewsに直しておいた方がいいかもしれないです。
```suggestion
export const News = () => Item_News;
```